src/support/azext_support/custom.py

```diff
@@ -17,12 +17,17 @@
 
 
 def list_support_tickets(cmd, client, filters=None):
+    logger.warning("Command group 'az support tickets' will be replaced by 'az support in-subscription tickets'" +
+                   " and 'az support no-subscription tickets'")
+    logger.warning("Parameter 'filters' argument will be replaced by 'filter'")
     if filters is None:
         filters = "CreatedDate ge " + str(date.today() - timedelta(days=7))
     return client.list(top=100, filter=filters)
 
 
 def get_support_tickets(cmd, client, ticket_name=None):
+    logger.warning("Command group 'az support tickets' will be replaced by 'az support in-subscription tickets'" +
```

Comment:
Please rerun your tests lively to ensure those warning messages won't break customers.

Reply:
Done